benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -6,576 +6,576 @@
-msgstr "DS Logon"
+msgid "ID.me"
+msgstr ""
 
-msgid "eligibility.pages.start.veteran.required_items[3]"
-msgstr "My HealtheVet"
+msgid "DS Logon"
+msgstr ""
 
-msgid "eligibility.pages.start.veteran.start_item.secondary_details"
+msgid "My HealtheVet"
 msgstr ""
+
+msgid ""
 "If you do not have an account with any of these services, you will need to "
 "create one. We recommend using Login.gov."
+msgstr ""
 
 msgctxt "image alt text"
-msgid "core.icons.idcardcheck"
-msgstr "Identification card icon with checkmark"
+msgid "Identification card icon with checkmark"
+msgstr ""
 
-msgid "core.modals.back"
-msgstr "Go back"
+msgid "Go back"
+msgstr ""
 
-msgid "eligibility.pages.index.senior.help.headline"
-msgstr "Learn more about"
+msgid "Learn more about"
+msgstr ""
 
 msgctxt "image alt text"
-msgid "core.logos.login_gov"
-msgstr "TODO"
-
-msgid "eligibility.pages.start.senior.help.headline"
-msgstr "Verify your identity with "
+msgid "TODO"
+msgstr ""
 
-msgid "eligibility.pages.index.mst_courtesy_card.label"
-msgstr "MST Courtesy Card"
+msgid "Verify your identity with "
+msgstr ""
 
-msgid "eligibility.pages.index.mst_courtesy_card.description"
+msgid "MST Courtesy Card"
 msgstr ""
+
+msgid ""
 "This option is for people who have a current Courtesy Card or an MST RIDES "
 "eligibility card."
+msgstr ""
 
-msgid "eligibility.pages.index.senior.label"
-msgstr "Older Adult"
-
-msgid "eligibility.pages.index.senior.description"
+msgid "Older Adult"
 msgstr ""
-"You must be 65 years or older. You will need to verify your identity with"
 
-msgid "eligibility.pages.index.veteran.label"
-msgstr "US Veteran"
+msgid ""
+"You must be 65 years or older. You will need to verify your identity with"
+msgstr ""
 
-msgid "eligibility.pages.index.veteran.description"
+msgid "US Veteran"
 msgstr ""
+
+msgid ""
 "This option is for people who have served in the active military, naval, or "
 "air service, and who were discharged or released therefrom under conditions "
-"other than dishonorable. You will need to <a href='https://www.va.gov/"
-"resources/verifying-your-identity-on-vagov/' target=\"_blank\" rel="
-"\"noopener noreferrer\">verify your identity through VA.gov</a>."
-
-msgid "eligibility.pages.index.intro.mst"
+"other than dishonorable. You will need to <a href=\"https://www.va.gov/"
+"resources/verifying-your-identity-on-vagov/\" target=\"_blank\" "
+"rel=\"noopener noreferrer\">verify your identity through VA.gov</a>."
 msgstr ""
-"Cal-ITP doesn’t save any of your information. All MST transit benefits "
-"reduce fares by 50% for buses on fixed routes."
 
-msgid "eligibility.pages.index.intro.sacrt"
+#, python-format
+msgid ""
+"Cal-ITP doesn’t save any of your information. All MST transit benefits "
+"reduce fares by 50%% for buses on fixed routes."
 msgstr ""
+
+#, python-format
+msgid ""
 "Cal-ITP doesn’t save any of your information. All SacRT transit benefits "
-"reduce fares by 50% for light rail."
+"reduce fares by 50%% for light rail."
```

Comment:
Double `%%` here needed to escape the ID, but gets formatted out of the final string.